code/game/say.dm

```diff
@@ -71,14 +71,35 @@ GLOBAL_LIST_INIT(freqtospan, list(
 	if(istype(D) && D.display_icon(src))
 		languageicon = "[D.get_icon()] "
 
-	return "[spanpart1][spanpart2][freqpart][languageicon][compose_track_href(speaker, namepart)][namepart][compose_job(speaker, message_language, raw_message, radio_freq)][endspanpart][messagepart]"
+	//NSV13 - added [compose_rank(speaker)]
+	return "[spanpart1][spanpart2][freqpart][languageicon][compose_track_href(speaker, namepart)][compose_rank(speaker)][namepart][compose_job(speaker, message_language, raw_message, radio_freq)][endspanpart][messagepart]"
```

Comment:
Doesn't compose_rank return GSGT Sergei Koralev, so then compose rank + namepart returns GSGT Sergei Koralev Sergei Koralev?

Unless i'm misreading this code

Reply:
Nah, compose_rank doesn't involve the name